examples/editor/src/App.tsx

```diff
@@ -1,14 +1,498 @@
 // import logo from './logo.svg'
+import { Block, Editor } from "@blocknote/core";
```

Comment:
this file needs to be cleaned up / reverted, or if it's useful, add a different example or test